python/tk_multi_publish2/dialog.py

```diff
@@ -398,9 +398,13 @@ def _push_settings_into_ui(self, selected_tasks):
         """
         # The run_get_ui_settings expects a dictionary of the actual values, not Setting objects, so
         # translate the dictionary.
-        tasks_settings = [
-            {k: v.value for k, v in task.settings.iteritems()} for task in selected_tasks
-        ]
+        tasks_settings = []
```

Comment:
Not sure why these changes are appearing again here, i cherry-picked them from your commit